onnxmltools/proto/__init__.py

```diff
@@ -4,17 +4,20 @@
 # license information.
 # --------------------------------------------------------------------------
 
+def _check_onnx_version():
+    import onnx
+    assert onnx.__version__ >= '1.0.1', 'ONNXMLTools requires ONNX-1.0.1 or newer version'
```

Comment:
Thanks for your comment. Could you share more context? Did you mean we should something like [this](https://github.com/Blosc/bcolz/commit/ef872dd052d4dda9ff516e67b719caccfffb1b2d)? Because all other files should access ONNX through this submodule, I personally think it's really a "before".

Reply:
I mean
```python
import pkg_resources
verTuple=pkg_resources.get_distribution("onnx").parsed_version.base_version.split(".")
```

Reply:
I will fix this. Thanks.